src/views/studio/MyVideosView/MyVideosView.tsx

```diff
@@ -199,6 +219,20 @@ export const MyVideosView = () => {
           <>
             <TabsContainer>
               <Tabs initialIndex={0} tabs={mappedTabs} onSelectTab={handleSetCurrentTab} />
+              <SortContainer>
+                <Text variant="body2">Sort by</Text>
+                <Select
+                  helperText={null}
+                  value={sortVideosBy}
+                  items={[...SORT_OPTIONS]}
```

Comment:
Let's remove that destructuring here. If needed let's remove `as const` as well, we can use the type of value directly (`VideoOrderByInput`)